modules/odc-viewing-application-topology.adoc

```diff
@@ -22,3 +22,5 @@ You can customize the views as required using the following:
 ** *Show* elements based on *Pod Count* or *Labels*
 *** Pod Count: Select to show the number of pods of a component in the component icon.
 *** Labels: Toggle to show or hide the component labels.
+
+The *Topology* view also provides you the *Export application* option to download your application in the ZIP file format. You can then import the downloaded application to another project or cluster. For more details, see _Exporting an application to another project or cluster_ in the _Additional resources_ section.
```

Comment:
Do you want to link to the `Exporting an application to another project or cluster`? Or do you simply want to direct users how to access this doc? 

When referring users to actual docs, I'm used to seeing those docs included as links or xrefs.

Reply:
@jc-berger, I guess cross-refernces or links are not supported in any module. This is why i provided the cross-refernce in the additional resources section of the assembly this module is a part of.

See [applications/odc-viewing-application-composition-using-topology-view.adoc](https://github.com/openshift/openshift-docs/pull/49524/files#diff-ee9adcdc33a181eba8d0ad16724b1fbb4b731fddce7a38422c9144bec1d9e26b) in the Files changed tab.